src/scaffold/app/nix.rs

```diff
@@ -9,6 +9,7 @@ use crate::error::{ScaffoldError, ScaffoldResult};
 use crate::file_tree::*;
 use crate::versions::holochain_nix_version;
 
+#[allow(clippy::obfuscated_if_else)]
```

Comment:
Is this a temporary thing pending improving the code? Otherwise a comment explaining the allow would be good :)

Reply:
I am a little indifferent to this lint rule, clippy recommends using

```rs
let value = if bool_var { "result" } else { "" };
```

instead of:

```rs
let value = bool_var.then_some("result").unwrap_or("");
```

I find the latter much cleaner and _subjectively_ more readable

Reply:
Fixed, clippy is satisfied with this instead

```rs
let value = bool_var.then_some("result").unwrap_or_default();
```

Reply:
It looks like `rust-clippy#9100` was only added for style reasons which are subjective as you say, but looking at this I'm thinking it's unlikely to compile to the same thing because it's going through an intermediate type to evaluate a boolean condition.

Doesn't seem like a big issue to me, I'd be fine with a comment on the lint rule pointing to this conversation. I do think it's useful to avoid somebody else having to figure out why ignoring it was reasonable by commenting though :)